src/lib/Microsoft.Health.Events/Telemetry/Metrics/EventMetricDefinition.cs

```diff
@@ -33,5 +33,7 @@ private EventMetricDefinition(string metricName)
         public static EventMetricDefinition DeviceIngressSizeBytes { get; } = new EventMetricDefinition(nameof(DeviceIngressSizeBytes));
 
         public static EventMetricDefinition MeasurementToFhirBytes { get; } = new EventMetricDefinition(nameof(MeasurementToFhirBytes));
+
+        public static EventMetricDefinition PartitionLastClaimedDelay { get; } = new EventMetricDefinition(nameof(PartitionLastClaimedDelay));
```

Comment:
Alternatively, if in the alert configuration we can include a check for how long since the delay metrics was last emitted that could serve the purpose too.  We will need to think about how to ensure we don't get false positives for services that are deprovisioned.